src/testerEmailParser.ts

```diff
@@ -0,0 +1,31 @@
+import * as fs from 'fs-extra';
```

Comment:
Since this is only used by app distribution, consider moving it into an app distribution specific directory instead of `/src`

Reply:
+1

Reply:
Its used by a couple different utility files.